plugins/plugins.gradle.kts

```diff
@@ -3,13 +3,6 @@ import org.gradle.jvm.tasks.Jar
 
 subprojects {
     afterEvaluate {
-        // Ensure that the WPILibVersioningPlugin is setup by setting the release type, if releaseType wasn't
-        // already specified on the command line
-        if (!hasProperty("releaseType")) {
-            WPILibVersion {
-                releaseType = ReleaseType.DEV
```

Comment:
Also need to remove the import for `ReleaseType` since it's no longer used. Running the `spotlessApply` task should fix it automatically